Gemfile

```diff
@@ -30,12 +30,20 @@ if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.6')
   gem 'simplecov-cobertura', '~> 2.1'
 end
 
-if ENV['REDIS_VERSION']
-  gem 'redis', "~> #{ENV['REDIS_VERSION']}"
+if redis_version = ENV.fetch('REDIS_VERSION', nil)
+  gem 'redis', "~> #{redis_version}"
```

Comment:
For some reason RuboCop was only forcing `fetch` in some cases below, so I just went ahead and updated all of them.